spec/lib/submission_builder/ty2024/states/md/md_return_xml_spec.rb

```diff
@@ -85,7 +87,7 @@
           it "attaches a 502R" do
             expect(xml.at("Form502R")).to be_present
             expect(instance.pdf_documents).to be_any { |included_document|
-              included_document.pdf == PdfFiller::Md502RPdf
+              included_document.pdf = PdfFiller::Md502RPdf
```

Comment:
looking at the rest of the file, looks like maybe just a typo!
-- 
why is the equal sign removed here? was this just a typo? I feel like the `==` comparison is necessary to satisfy `be_any` matcher, but I can't find good docs on this. I only found:
https://gist.github.com/mcmire/5860047

Reply:
tbh i was just trying to follow the existing pattern which seemed to be `=` for `.to be_any` and `==` for `.not_to be_any`.

i have no idea how `be_any` works though and as pointed out in that gist, it doesn't really grammatically make sense. what i've figured out from trial and error is that both `==` and `=` work for the positive case (`.to be_any`) and only `==` works for the negative case (`.not_to be_any`). so we could either leave it as is or change everything to `==`

Reply:
actually please ignore that i think i maybe understand now why `=` was passing for the positive case? because setting `.pdf` returns true?

anyway changing them all to `==`